ecl/eclagent/eclagent.cpp

```diff
@@ -1456,7 +1456,7 @@ bool EclAgent::fileExists(const char *name)
     StringBuffer lfn;
     expandLogicalName(lfn, name);
 
-    Owned<IDistributedFile> f = queryDistributedFileDirectory().lookup(lfn.str(),queryUserDescriptor());
+    Owned<IDistributedFile> f = queryDistributedFileDirectory().lookup(lfn.str(),queryUserDescriptor(), false, false, false, nullptr, defaultPrivilegedUser);
```

Comment:
I know this is not reading content, but is there any benefits from passing defaultPrivilegedUser here, when isPrivilegedUser is available?

Reply:
actually forget that, fileExists() is only used internally.